docs/drivers/aws.md

```diff
@@ -141,3 +143,8 @@ To create a machine with a non-default vpc-id:
     $ docker-machine create --driver amazonec2 --amazonec2-access-key AKI******* --amazonec2-secret-key 8T93C********* --amazonec2-vpc-id vpc-****** aws02
 
 This example assumes the VPC ID was found in the `a` availability zone. Use the`--amazonec2-zone` flag to specify a zone other than the `a` zone. For example, `--amazonec2-zone c` signifies `us-east1-c`.
+
+## Custom AMI and ssh user name
+The ssh user name must match the one on your ami, the defaults ssh user name, matches the default ami we use.
```

Comment:
"The SSH username must match the one on your AMI.  The default SSH username matches the default AMI that Machine uses."